src/scss/_themes.scss

```diff
@@ -63,7 +66,7 @@
 				padding: 0 10px;
 			};
 
-			&:nth-child(4) {
+			&:last-child {
```

Comment:
This is true. I guess if you had never seen the output of the code it would make more sense to read last rather than fourth?